internal/multicluster/internal/controllers/v1compat/controller.go

```diff
@@ -87,7 +91,9 @@ func (r *reconciler) Reconcile(ctx context.Context, rt controller.Runtime, req c
 	entMeta.OverridePartition(req.ID.Tenancy.Partition)
 	existing, err := r.config.GetExportedServicesConfigEntry(ctx, req.ID.Tenancy.Partition, entMeta)
 	if err != nil {
-		rt.Logger.Error("error getting exported service config entry", "error", err)
+		// When we can't read the existing exported-services we purposely allow
+		// reconciler to continue so we can still write a new one
+		rt.Logger.Warn("error getting exported service config entry but continuing reconcile", "error", err)
```

Comment:
nit: I know we do this all over the place but in general it seems to be more "go-like" to not include the word `error` in the error string. I think partially just since this makes for a cleaner message when multiple errors are wrapped up together. Else we end up with something like `error doing foo: error doing bar: error doing baz`

https://go.dev/wiki/CodeReviewComments#error-strings